### ISSUE TITLE
Initial data / data example?

### ISSUE THREAD

**Issue by bagage** [flightlessmango/flightlessmango.com#36]
I'm trying to host an instance locally, but yet my database is empty and it seems the app expects some data to be already present (for instance games with source `mango`).

Is there a way to have a working local copy?

I tried to manually create these data (and probably doying it wrong) without success yet:

```
Api.create(name: "api-test")
Game.create(name: "game1", slug:"game1", source:"mango")
Type.create(name: "type1")
Variation.create(name: "var1", type_id: 1)
```

Thanks!

**Comment by flightlessmango** [comment]
Looks like you've basically got it but there is a formatting bug in `config/storage.yml` which will prevent uploading, replace it's contents with
https://raw.githubusercontent.com/huacnlee/rails-activestorage-example/master/config/storage.yml

Game just needs a name and source mango like so
`Game.create!(name: "VideoGame1", source: "mango")`

I don't use HML or OCAT anymore so I'm not a 100% that they are still working as intended

**Comment by bagage** [comment]
Yes indeed I have trouble with uploading, `undefined method `upload' for nil:NilClass`. Even with the updated `storage.yml` it is failing but not sure why. Any idea would be greatly appreciated :+1:.

In the mean time some questions:

- Could you confirm me that http://localhost:3001/games/1/logs/new log upload expects a CSV file like [this one](https://gist.github.com/bagage/ca2201a1e695f112f8eba8f35032f78b)?
- On http://localhost:3001/benchmarks/youtubeid/edit, what does the MANGO file should look like? Is it a CSV again (the same one from previous point)?

**Comment by flightlessmango** [comment]
As I said there is a formatting error in the `config/storage.yml` that is in this repo, you need to replace it with this one
https://raw.githubusercontent.com/huacnlee/rails-activestorage-example/master/config/storage.yml to fix uploading

Yes the file that user benchmarks expects and a "MANGO" file are the same, a csv created by mangohud
